src/Driver/OCI8/Connection.php

```diff
@@ -47,22 +47,30 @@ public function getServerVersion(): string
         return $matches[1];
     }
 
-    /** @throws Parser\Exception */
+    /**
+     * @throws Parser\Exception
+     * @throws Error
+     */
     public function prepare(string $sql): Statement
     {
         $visitor = new ConvertPositionalToNamedPlaceholders();
 
         $this->parser->parse($sql, $visitor);
 
-        $statement = oci_parse($this->connection, $visitor->getSQL());
+        $statement = @oci_parse($this->connection, $visitor->getSQL());
         assert(is_resource($statement));
```

Comment:
Shouldn't we remove this line now?

Reply:
wasn't sure if the assert should be removed, so I kept it consistent with the PostgreSQL driver by adding the error check alongside it. But can remove it if there is no reason for it to be here anymore?

Reply:
> if there is no reason for it to be here anymore

I don't see any. 🤷‍♂️ Let's remove it.

Reply:
Turned out it had to go anyway, assert is enabled when tests run 😄